src/Screens/All.tsx

```diff
@@ -20,19 +25,20 @@ const All = ({ navigation }: Props) => {
   );
 };
 
-const styles = StyleSheet.create({
-  logoImage: { flex: 1, height: "100%" },
-  logoContainer: {
-    height: 100,
-    justifyContent: "center",
-    alignItems: "center",
-    marginHorizontal: 24,
-    marginVertical: 24,
-  },
-  container: {
-    flex: 1,
-    backgroundColor: "#fff",
-  },
-});
+const genStyles = (colors: ExtendedTheme["colors"]) =>
```

Comment:
Checkout some library that support dark/light mode. `StyleSheet.create` has some limitations. Maybe: https://github.com/jpudysz/react-native-unistyles ?

Reply:
If you want to stay with your solution, then you should create some hook/style wrapper that does all magic.

Reply:
Ok, I decided to do my solution because it seems to be enough for this app size, but go through this library seems to be nice experience but I will move all spacing sizing etc to thema

Reply:
I will stay with my solution, unistyles demanding prebuild my expo app